Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs

```diff
@@ -166,7 +170,14 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
                 false,
                 true);
 
+            var totalFailedTestsCount = this.ExtractTotalFailedTestsCount(processExecutionResult.ReceivedOutput);
```

Comment:
Can you add a check that the number of total tests caught from the TestResultsRegex is equal to the number of Tests in the Execution Contex.